velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -114,6 +114,24 @@ TEST_F(DateTimeFunctionsTest, weekOfYear) {
   EXPECT_EQ(15, weekOfYear("2013-04-08"));
 }
 
+TEST_F(DateTimeFunctionsTest, unixDate) {
+  const auto unixDate = [&](const StringView& date) {
```

Comment:
const StringView& -> std::string_view

use std::string_view and pass it by value

Reply:
Updated.